supervisor.go

```diff
@@ -36,6 +39,26 @@ func (s *Supervisor) Run() {
 		configureSysProcAttr(s.cmd)
 		s.cmd.Env = append(os.Environ(), s.Options.Env...)
 
+		if s.Options.User != "" {
+			currentUser,_ := user.Current()
+
+			if currentUser.Username != s.Options.User {
+				executingUser, err := user.Lookup(s.Options.User)
+
+				if err != nil {
+					s.logger.Error("unknown user: ", zap.Error(err))
```

Comment:
Maybe this check should be done here https://github.com/Baldinof/caddy-supervisor/blob/master/definition.go#L59

So it will not try to start with an invalid user.

Reply:
done